lib/metrics/events.js

```diff
@@ -98,7 +100,12 @@ module.exports = log => {
         }
 
         return emitFlowEvent(event, request, data)
-          .then(() => {})
+      })
+      .then(metricsContext => {
+        amplitude(event, request, data, metricsContext)
+        if (metricsContext && event === metricsContext.flowCompleteSignal) {
+          amplitude('flow.complete', request, data, metricsContext)
```

Comment:
Is this so that the completion event gets emitted in the correct order?  It may be worth a brief comment, I was surprised to see the "if event === flowCompleteSignal" duplicated both here and in the `emitFlowEvent` function.